time stamp
spinal/ServoState[] servos
